covariate-shift/covariate-shift.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "Covariate Shift",
+    "description": "Indicates whether there is a data distribution shift between two datasets. Tries to predict whether a data point came from a 'Training' dataset or 'Production' dataset.",
+    "inputs": [
+        {
+            "name": "train-dst",
+            "type": "dataset-id",
+            "description": "Training dataset. (Dataset we may have used to train a model)"
+        },
+        {
+            "name": "prod-dst",
+            "type": "dataset-id",
+            "description": "Production dataset. (Dataset that represents current data. I.e., data on which we want to make predictions)"
+        },
+        {
+            "name": "n",
+            "type": "number",
+            "description": "Number of phi coefficients to calculate",
+            "default": 5
+        }
+    ],
+    "outputs": [
+        {
+            "name": "output",
+            "type": "map",
+            "description": "Map of results. Displays both the average of all individual phi coeffiencet runs and the average across all runs."
+        }
+    ]
+}
```

Comment:
this file is miising "kind" and "source_code", and be called "metadata.json".  likewise, the script should be named "script.whizzml".  see any other script in this collection for examples.